extensions/iconify/src/search-icons.tsx

```diff
@@ -49,7 +49,7 @@ function Command() {
       {icons.map((icon) => {
         const { set, id, body, width, height } = icon;
         const { id: setId, title: setName } = set;
-        const svgIcon = toSvg(body, width, height, iconColor);
+        const svgIcon = toSvg(body, width, height, iconColor === iconColorEnum.customColor && customColor ? customColor : iconColor);
```

Comment:
The customColor must include a check to verify whether the color format is correct; otherwise, it will cause an icon error.